experimental/codepointtrie/src/lib.rs

```diff
@@ -0,0 +1,9 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
```

Comment:
Issue: Please add lots of docs tests.  You can use the `.get_planes_trie()` as a starting point for most of them.  You can save this for later if you like, but we need docs tests before merging to components.

Reply:
Agreed, there need to be more documentation and doc tests. Would like to save this for later, before merging to components.

Reply:
OK, just make sure then that this is tracked somewhere so we don't forget.